### PR TITLE
Vp2420 minimal regression fixes

### PR DESCRIPTION
**Pull request by macpijan**
@mkopec @filipleple can it break anything for laptops case?

### CODE REVIEW

**Comment by macpijan** [comment]
Result from minimal regression so far: https://paste.dasharo.com/?6c2b1d03dfe0baec#2s9NXsoGVA8azbQ4BMg1bpyRDznhjXJ6HCZhJaqDzbV5

Comment:
- USB keyboard - expected, not connected - I still think it's not the best case as it requires to manually connect physical keyboard to the platform each time we want to run the test
- WiFi - need checking, do we even have a card there?
- STB002.001 - caused by the incorrect printk settings in the OS, some details here: https://github.com/Dasharo/open-source-firmware-validation/pull/449/files; will rerun again

**Comment by macpijan** [comment]
After setting:

```
cat /etc/sysctl.d/10-console-messages.conf 

# the following stops low-level messages on console
kernel.printk = 0 4 1 7
```

`STB002.001` passed as well

**Comment by filipleple** [comment]
This will make 
```
        Login To Linux
        Switch To Root User
```
execute twice in a row on laptops, but I would guess doing this over SSH will be properly handled and won't cause any problems

**Comment by macpijan** [comment]
Can you elaborate @filipleple  ? Can you please explain where does the second one come from?

**Comment by filipleple** [comment]
@macpijan sorry, I must've been looking at some older revision, I checked the `Boot System...` keyword again and it should align nicely with the changes to `lib/flash`, causing no trouble with laptops.

File: platform-configs/protectli-vp2420.robot [hunk 1]
```diff
@@ -9,9 +9,9 @@ ${FLASHING_METHOD}=                 internal
 # eMMC driver support
 ${E_MMC_NAME}=                      8GTF4R
 ${DMIDECODE_SERIAL_NUMBER}=         N/A
-${DMIDECODE_FIRMWARE_VERSION}=      Dasharo (coreboot+UEFI) v1.2.0
+${DMIDECODE_FIRMWARE_VERSION}=      Dasharo (coreboot+UEFI) v1.2.1-rc1
 ${DMIDECODE_PRODUCT_NAME}=          VP2420
-${DMIDECODE_RELEASE_DATE}=          10/12/2023
+${DMIDECODE_RELEASE_DATE}=          18/12/2024
```
Comment (wiktormowinski) [inline platform-configs/protectli-vp2420.robot]:
```
DMI004.001 Verify the firmware release date :: Check whether the f... | FAIL |
'# dmidecode 3.3
Getting SMBIOS data from sysfs.
SMBIOS 3.3.0 present.

Handle 0x0000, DMI type 0, 26 bytes
BIOS Information
        Vendor: 3mdeb
        Version: Dasharo (coreboot+UEFI) v1.2.1-rc1
        Release Date: 12/18/2024
        ROM Size: 16 MB
        Characteristics:
                PCI is supported
                PC Card (PCMCIA) is supported
                BIOS is upgradeable
                Selectable boot is supported
                ACPI is supported
                Targeted content distribution is supported
                UEFI is supported
        BIOS Revision: 1.2
        Firmware Revision: 0.0' does not contain 'Release Date: 18/12/2024'

```
it looks like you added date in european format and the bios info has this date in american

Comment (wiktormowinski) [reply platform-configs/protectli-vp2420.robot]: https://github.com/Dasharo/open-source-firmware-validation/pull/643/commits/96b0c24f5102d0e00927719750f2729019c791f3

Comment (macpijan) [reply platform-configs/protectli-vp2420.robot]: thanks

**Review by wiktormowinski — APPROVED** [review]

**Pull request merged**